numpy/core/tests/test_shape_base.py

```diff
@@ -191,6 +191,16 @@ def test_2D_array2(self):
 
 
 class TestConcatenate(object):
+    def test_returns_copy(self):
+        a = np.eye(3)
+        b = np.concatenate([a])
+        b[0, 0] = 2
+        assert b[0, 0] != a[0, 0]
+
+        b = np.concatenate(a)
```

Comment:
This isn't really a good test - unlike `block`, `concatenate` isn't designed to take a single array as an argument.

Reply:
ok.